Tools/scripts/summarize_stats.py

```diff
@@ -28,6 +27,15 @@ def format_ratio(num, den):
     else:
         return f"{num/den:.01%}"
 
+def ratio_to_float(s):
+    """
+    The inverse of format_ratio.
+    """
+    if s == "":
+        return 0.0
+    else:
+        return float(s[:-1])
+
```

Comment:
This confused me, because it's not the inverse, right? It just turns `"4.2%"` into `4.2` (which *feels* wrong to me since I'd expect `.042`... I know that this is what the old code does, and it's only used as a sort key, so I guess either is fine).

So I'd suggest changing the name and docstring, but I honestly can't think of a good name for this operation...

At least, if this is gonna be its own function, maybe `assert s.endswith("%")` so it's clear what this function is doing?

Reply:
Good point.  I renamed the function and clarified in the docstring.